js/main.js

```diff
@@ -16,17 +16,137 @@ if (previousEntries) {
 $form.addEventListener('submit', function (e) {
   e.preventDefault();
   data.profile.avatarUrl = $form.elements.avatarUrl.value;
-  data.profile.username = $form.elements.username.value;
+  data.profile.username = $form.elements.username.value.trim();
   data.profile.fullName = $form.elements.fullname.value;
   data.profile.location = $form.elements.location.value;
   data.profile.bio = $form.elements.bio.value;
 
   data.entries.push(data.profile);
```

Comment:
I must have missed this before, but you shouldn't be using `data.entries` for this, that will be used in the entry features ahead. You should just be changing the values stored in `data.profile`